### PR TITLE
status_ipsec.php: show encr-keysize

### PR DESCRIPTION
**Pull request by vktg**
- [ ] Redmine Issue: https://redmine.pfsense.org/issues/9943
- [ ] Ready for review

Show size of selected encryption algo on Status \ IPsec page
without it, AES-GCM 128/192/256 is always displayed as AES_GCM_16,
instead of AES_GCM_16_128/AES_GCM_16_192/AES_GCM_16_256

'ipsec status' command format

### CODE REVIEW

File: src/usr/local/www/status_ipsec.php [hunk 1]
```diff
@@ -214,6 +214,9 @@ function print_ipsec_body() {
 			print("</td>\n");
 			print("<td>\n");
 			print(htmlspecialchars($ikesa['encr-alg']));
+			if ($ikesa['encr-alg'] != '3DES_CBC') {
+                                print('_' . htmlspecialchars($ikesa['encr-keysize']));
+                        }
```
Comment (rbgarga) [inline src/usr/local/www/status_ipsec.php]: Fix indent, using TABs

File: src/usr/local/www/status_ipsec.php [hunk 2]
```diff
@@ -339,7 +342,11 @@ function print_ipsec_body() {
 					print("</td>\n");
 					print("<td>\n");
 
-					print(htmlspecialchars($childsa['encr-alg']) . '<br/>');
+					print(htmlspecialchars($childsa['encr-alg']));
+                                        if ($childsa['encr-alg'] != '3DES_CBC') {
+                                                print('_' . htmlspecialchars($childsa['encr-keysize']));
+                                        }
+                                        print('<br/>');
```
Comment (rbgarga) [inline src/usr/local/www/status_ipsec.php]: Fix indent, using TABs

File: src/usr/local/www/status_ipsec.php (hunk 1 above)
Comment (vktg) [reply src/usr/local/www/status_ipsec.php]: done

File: src/usr/local/www/status_ipsec.php (hunk 2 above)
Comment (vktg) [reply src/usr/local/www/status_ipsec.php]: done

File: src/usr/local/www/status_ipsec.php [hunk 3]
```diff
@@ -339,7 +342,11 @@ function print_ipsec_body() {
 					print("</td>\n");
 					print("<td>\n");
 
-					print(htmlspecialchars($childsa['encr-alg']) . '<br/>');
+					print(htmlspecialchars($childsa['encr-alg']));
+					if ($childsa['encr-alg'] != '3DES_CBC') {
+						print('_' . htmlspecialchars($childsa['encr-keysize']));
+					}
+                                        print('<br/>');
```
Comment (rbgarga) [inline src/usr/local/www/status_ipsec.php]: This line is still using spaces

Comment (vktg) [reply src/usr/local/www/status_ipsec.php]: it's already in the last snapshot:
php`if (!empty($childsa['encr-keysize'])) {
          print(" (" . htmlspecialchars($childsa['encr-keysize']) . ")");
}
print('<br/>');`

PR can be closed

**Pull request closed without merging**